contracts/src/CrossChainApplications/examples/NativeTokenBridge/tests/NativeTokenDestinationTests.t.sol

```diff
@@ -139,6 +145,17 @@ contract NativeTokenDestinationTest is NativeTokenBridgeTest {
             )
         );
 
+        vm.expectEmit(true, true, true, true, address(nativeTokenDestination));
```

Comment:
Can't comment on the exact line, but I'm confused why the `expectCall` to the native minter precompile with a value of 0 is passing. Shouldn't we not call the precompile if the amount is 0?

Reply:
Aha! the 0 there is "expect 0 calls", nice. Could you add a comment clarifying that, I read it as "expect a call with a value of 0"